tests/snappi/test_vlan_traffic.py

```diff
@@ -0,0 +1,169 @@
+import pytest
+from pprint import pprint as pprint
+import time
+
+vlan_id = "10"
+macs = ["00:1A:C5:00:00:01", "00:1B:6E:00:00:01"]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def discovery(npu):
+    npu.objects_discovery()
+
+
+def test_l2_traffic(npu, dataplane):
+    """
+    Creates vlan 10 and adds two ports to the vlan 10 member
+    and validates the config using l2 traffic
+    """
+    commands = [
+        {
+            "name": "vlan_10",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN",
+            "attributes": ["SAI_VLAN_ATTR_VLAN_ID", "10"],
+        },
+        {
+            "name": "vlan_member_2",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN_MEMBER",
+            "attributes": [
+                "SAI_VLAN_MEMBER_ATTR_VLAN_ID",
+                "$vlan_10",
+                "SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID",
+                "$BRIDGE_PORT_2",
+                "SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE",
+                "SAI_VLAN_TAGGING_MODE_UNTAGGED",
+            ],
+        },
+        {
+            "name": "PORT_2",
+            "op": "set",
+            "type": "SAI_OBJECT_TYPE_PORT",
+            "attributes": [
```

Comment:
I'm surprised it works. In SAI, the set operation allows to set one attribute at a time. So, I suspect that the second attribute is actually ignored. Please check. Probably we should file a bug..

Reply:
created issue https://github.com/opencomputeproject/SAI-Challenger/issues/174 for this